app_modules/ui/svg-icon/index.jsx

```diff
@@ -21,18 +21,18 @@ class SvgIcon extends React.Component {
   render() {
     const className = this.$getClassName();
     const props = this.$propsWithoutKeys('className', 'sprite', 'symbol');
+    const {sprite,symbol} = this.props;
```

Comment:
Same as for previous pull request #24 ...
Before we can merge the Pull Request, would you mind do the following steps:
https://github.com/salesforce-ux/design-system/blob/winter-16/CONTRIBUTING.md

Thank you again.

Bruno